client/app.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// Declare app level module which depends on views, and components
+angular.module('myApp', [
+  'ngRoute',
```

Comment:
Since we will be using hugo for defining routes and since not all the site will be within angular (it's ooni-web) I would advise to not use routes, but to just define controllers and hook the controllers directly in the HTML of the page.